webservice/dispatcher.php

```diff
@@ -34,6 +34,11 @@
 Context::getContext()->cart = new Cart();
 Context::getContext()->container = ContainerBuilder::getContainer('webservice', _PS_MODE_DEV_);
 
+// If the currency is not defined in the context, initialize with the default one !
```

Comment:
Hi @thalassa-web I believe this line is actually part of another PR, right ? It's not linked to this topic 😉

Reply:
Yes you're right … Sorry …